djstripe/models.py

```diff
@@ -19,15 +19,16 @@
 import stripe
 
 from . import settings as djstripe_settings
-from .exceptions import SubscriptionCancellationFailure
+from .exceptions import SubscriptionCancellationFailure, SubscriptionApiError
 from .managers import CustomerManager, ChargeManager, TransferManager
 from .signals import WEBHOOK_SIGNALS
 from .signals import subscription_made, cancelled, card_changed
 from .signals import webhook_processing_error
+from .settings import ALLOW_MULTIPLE_SUBSCRIPTIONS, RETAIN_CANCELED_SUBSCRIPTIONS
 
 
 stripe.api_key = settings.STRIPE_SECRET_KEY
-stripe.api_version = getattr(settings, "STRIPE_API_VERSION", "2012-11-07")
+stripe.api_version = getattr(settings, "STRIPE_API_VERSION", "2014-01-31")
```

Comment:
Careful with this - I have yet to go through and check api changes (see #162). Is there a reason you need this version of the API?